src/components/ui/menu-option/menu-option.scss

```diff
@@ -34,21 +28,21 @@ $menu-item-padding: 12px;
   padding: 0 $sidebar-padding 0 calc(#{$sidebar-padding} - 2px);
   user-select: none;
   white-space: nowrap;
+  border-bottom: solid var(--menu-option-border-color);
```

Comment:
Hey @GabrielComymQB, can you please check it and confirm if the border style is correct for each scenario (active/hover/unselected) ? Thank you

Reply:
Hi!

I guess you're talking about the dividers that go at the bottom of each cell right?
That's a great question, cause we've been using a white colour with 10% transparency. However to make the divider have a universal colour regardless of the selection state, we can make the divider a solid colour #21333D

![Screenshot 2022-06-28 at 15 44 09](https://user-images.githubusercontent.com/55446576/176208461-bb7f94a3-1a2f-4e33-810f-ec5bf06a0c09.jpg)
d

Reply:
Thanks, I will add this new color.